### PR TITLE
Got rid of PS pattern synonym

### PR DESCRIPTION
**Pull request by Vlix**
Since `base16` now relies on `bytestring >= 0.11`, we can stop using the offset from `(PS _ offset _)` and just use the `BS` constructor.

### CODE REVIEW

File: src/Data/ByteString/Lazy/Base16.hs [hunk 1]
```diff
@@ -138,7 +138,7 @@ decodeBase16Lenient :: ByteString -> ByteString
 decodeBase16Lenient = fromChunks
   . fmap B16.decodeBase16Lenient_
   . reChunk
-  . fmap (B.filter (flip elem "0123456789abcdefABCDEF"))
+  . fmap (B.filter (`elem` "0123456789abcdefABCDEF"))
```
Comment (emilypi) [inline src/Data/ByteString/Lazy/Base16.hs]: unrelated and unnecessary change. Please revert.

**Review by emilypi — COMMENTED** [review]
Thanks, this is fine. Please address the comments and I'm fine with merging.

**Comment by Vlix** [comment]
Addressed the comment

**Review by emilypi — APPROVED** [review]
thanks!
